docs/conf.py

```diff
@@ -20,6 +20,9 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath(".."))
 
+# For plugins that can not read conf.py.
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
Is this a workaround for a `sphinx-docfx-yaml` bug?  Should we link that issue here?

Reply:
I will add it in the template.